docs/src/tutorial.md

```diff
@@ -7,13 +7,14 @@ Here are some tutorials on how to use Tulipa.
 For our first example, let's use a very small existing dataset.
 Inside the code for this package, you can find the folder [`test/inputs/Tiny`](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/tree/main/test/inputs/Tiny), which includes all the files necessary to create a TulipaEnergyModel and solve it.
 
-There are 8 files inside the "Tiny" folder. They define the assets and flows data, their profiles, and their time resolution.
-Furthermore, there are two files to define the representative periods and which periods in the full problem formulation they stand for.
-For more details on what are these files mean, see [Input](@ref)
+There are 8 relevant*files inside the "Tiny" folder. They define the assets and flows data, their profiles, and their time resolution, as well as two files to define the representative periods and which periods in the full problem formulation they stand for.\
+**Ignore the 9th file, bad-assets-data.csv, which is used for testing.*
```

Comment:
```suggestion
There are 8 relevant[^1] files inside the "Tiny" folder. They define the assets and flows data, their profiles, and their time resolution, as well as two files to define the representative periods and which periods in the full problem formulation they stand for.

[^1]: Ignore the 9th file, bad-assets-data.csv, which is used for testing.
```

Footnotes are available on GitHub[^1]. They might need an extension to render on VSCode.

[^1]: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes